### PR TITLE
fix: remove /node_modules/. from ignored paths

### LINKED ISSUE #183: Worker dependency issue
Looks like there is still a linger issue related to missing dependencies. Here are the repro steps.

Clone https://github.com/aspect-build/bazel-examples

```
cd jest
git checkout open_wc_repro_part_1
bazel clean
bazel build //src:src_test_ts
```
should fails with
```
error TS2307: Cannot find module '@open-wc/dedupe-mixin' or its corresponding type declarations.
```
since there is a missing transitive dependency in an npm package. Now

```
git checkout open_wc_repro_part_2
bazel build //src:src_test_ts
```
should still fail even tho the transitive dependency is now added
```
bazel shutdown
bazel build //src:src_test_ts
```
should now build successfully

### PR DESCRIPTION
**Pull request by thesayyn**
fixes #183

### CODE REVIEW

File: ts/private/ts_project_worker.js [hunk 1]
```diff
@@ -3,8 +3,14 @@ const ts = require('typescript');
 const path = require('path');
 const worker = require('@bazel/worker');
 const v8 = require('v8');
+
 const MNEMONIC = 'TsProject';
 
+// Workaround for https://github.com/microsoft/TypeScript/pull/42095
```
Comment (alexeagle) [inline ts/private/ts_project_worker.js:9]: why do we need a workaround if that PR was merged almost 2 years ago?

Comment (thesayyn) [reply ts/private/ts_project_worker.js:9]: these changes are the cause of the bug, not the solution.

Comment (alexeagle) [reply ts/private/ts_project_worker.js:9]: ah, then how about "workaround the issue introduced in ..." and ideally also link to (or create) an issue describing what they broke

**Review by alexeagle — APPROVED** [review]

File: ts/private/ts_project_worker.js (hunk 1 above)
Comment (thesayyn) [reply ts/private/ts_project_worker.js:9]: that's better. I'll file the issue with no-hope-to-see-it-fixed.

**Comment by gregmagolan** [comment]
Looks like you have to try/catch on the lstat to handle the file does not exist `ENOENT` case?

```
==================== Test output for //ts/test/ts_project_worker:fstree_test:
node:internal/fs/utils:344
    throw err;
    ^

Error: ENOENT: no such file or directory, lstat '/home/runner/.cache/bazel/_bazel_runner/537fa4813b44af3837a26a2cda63cb2b/sandbox/linux-sandbox/417/execroot/aspect_rules_ts/_tmp/4[58](https://github.com/aspect-build/rules_ts/actions/runs/3430459081/jobs/5717451300#step:6:59)5122692a90866cc108a43b4f7d3a8/tree/subtree/input.js'
    at Object.lstatSync (node:fs:1514:3)
    at Object.lstatSync (/home/runner/.cache/bazel/_bazel_runner/537fa4813b44af3837a26a2cda63cb2b/external/aspect_rules_js/js/private/node-patches/fs.js:110:23)
    at Object.add (/home/runner/.cache/bazel/_bazel_runner/537fa4813b44af3837a26a2cda63cb2b/sandbox/linux-sandbox/417/execroot/aspect_rules_ts/bazel-out/k8-fastbuild/bin/ts/test/ts_project_worker/fstree_test.sh.runfiles/aspect_rules_ts/ts/test/ts_project_worker/ts_project_worker.js:104:25)
    at Object.<anonymous> (/home/runner/.cache/bazel/_bazel_runner/537fa4813b44af3837a26a2cda[63](https://github.com/aspect-build/rules_ts/actions/runs/3430459081/jobs/5717451300#step:6:64)cb2b/sandbox/linux-sandbox/417/execroot/aspect_rules_ts/bazel-out/k8-fastbuild/bin/ts/test/ts_project_worker/fstree_test.sh.runfiles/aspect_rules_ts/ts/test/ts_project_worker/fstree.test.js:16:6)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -2,
  syscall: 'lstat',
  code: 'ENOENT',
  path: '/home/runner/.cache/bazel/_bazel_runner/537fa4813b44af3837a26a2cda63cb2b/sandbox/linux-sandbox/417/execroot/aspect_rules_ts/_tmp/4585122692a908[66](https://github.com/aspect-build/rules_ts/actions/runs/3430459081/jobs/5717451300#step:6:67)cc108a43b4f7d3a8/tree/subtree/input.js'
}
```

**Pull request merged**